packages/eks-controller/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: eks-controller
+description: A Helm chart for provisioning EKS clusters
+home: https://github.com/rancher/system-charts/charts/eks-controller
```

Comment:
I have a broader question if we should actually keep these charts in here that we are maintaining in this repo or if it should be a new/different one as these are source charts that aren't used in the old UI.

Reply:
Home and source should probably be where your source code is at. https://github.com/rancher/eks-controller